workflows/workflow/cp_process_singlecells.smk

```diff
@@ -34,6 +35,7 @@ configfile: "./configs/wf_configs/cp_process_singlecells.yaml"
 # importing modules
 include: "../rules/common.smk"
 include: "../rules/cytotable_convert.smk"
+include: "../rules/annotate.smk"
```

Comment:
Looking at line 30-32: how do multiple config files know which rules they belong to?

Reply:
Each module represented by the `include` takes in the same workflow config.

Here's an example from: https://github.com/WayScience/CytoSnake/blob/aa852f9f4886057cb96f5aa19e648e2d760adc22/configs/wf_configs/cp_process_singlecells.yaml

```yaml
cytotable_convert:
  params:
    dest_datatype: parquet
    source_datatype: sqlite
    concat: True
    join: True
    infer_common_schema: True
    drop_null: True
    preset: cellprofiler_sqlite
    log_level: ERROR

annotate_configs:
  params:
    join_on:
      - Metadata_well_position
      - Image_Metadata_Well
    add_metadata_id_to_platemap: True
    format_broad_cmap: False
    clean_cellprofiler: True
    external_metadata: "none"
    external_join_left: "none"
    external_join_right: "none"
    compression_options:
      method: "gzip"
      mtime: 1
    float_format: null
    cmap_args: {}
```

The corresponding parameters are selected by the name of the module. Above presents the parameters for both:

```yaml 
include: "../rules/cytotable_convert.smk"
include: "../rules/annotate.smk"
```

Here's an simple image:
 
![Screenshot 2023-05-19 at 7 14 10 AM](https://github.com/WayScience/CytoSnake/assets/31600622/ac01d408-bb16-4167-b74c-c8f92f8afd4f)